### ISSUE TITLE
Proposals viewer takes about 10 seconds to load

### ISSUE THREAD

**Issue by fire** [godot-proposals-viewer/godot-proposals-viewer.github.io#11]
The page should be fully loading within 250ms or 1 second.

**Comment by Calinou** [comment]
Unfortunately, I don't see a way this could be done. Pagination would be against the goal of this proposal viewer (which is to show everything on the same page for quick searching via <kbd>Ctrl + F</kbd>).

Loading split JSON files via chunks might work, but it will also mean that not all proposals will be displayed at first, which is misleading.

I can look into adding a proper progress bar, but I don't know if Alpine.js exposes enough information for that.

**Comment by gknzby** [comment]
Can I work on this issue? It looks like this issue still persist.

**Comment by Calinou** [comment]
> Can I work on this issue? It looks like this issue still persist.

Yes, but note that CI currently fails due to reaching token limits. I need to switch the backend fetcher to use GitHub's GraphQL API instead of the REST API (as it has more generous usage limits).

**Comment by Calinou** [comment]
Loading is much faster thanks to https://github.com/godot-proposals-viewer/godot-proposals-viewer.github.io/pull/35 (which limits the number of visible proposals to 200 by default), closing.
